superset-frontend/package.json

```diff
@@ -67,7 +67,8 @@
     "prune": "rm -rf ./{packages,plugins}/*/{lib,esm,tsconfig.tsbuildinfo,package-lock.json}",
     "storybook": "cross-env NODE_ENV=development BABEL_ENV=development storybook dev -p 6006",
     "tdd": "cross-env NODE_ENV=test jest --watch",
-    "test": "cross-env NODE_ENV=test jest",
+    "test": "cross-env NODE_ENV=test jest --max-workers=50%",
+    "test:ci": "cross-env NODE_ENV=test jest",
```

Comment:
What's the need for the additional command?

Reply:
Running `jest test` without setting workers in local machine could render plenty of your application (browser, IDE, et cetera) to be unresponsive and even cause some tests to timeout. I think that should be run in CI only

Reply:
It's N=1 but I've got the above case. Not sure if it's due to Jest upgrade or not (my hunch saying yes)